aten/src/ATen/native/Pool.h

```diff
@@ -120,7 +120,7 @@ inline std::pair<c10::SymInt, c10::SymInt> pooling_same_mode_padding_lr(
 inline void
 pool2d_shape_check(
   const Tensor& input,
-  int kH, int kW, int dH, int dW, int padH, int padW, int dilationH, int dilationW,
+  int64_t kH, int64_t kW, int64_t dH, int64_t dW, int64_t padH, int64_t padW, int64_t dilationH, int64_t dilationW,
```

Comment:
Why does dilationH, dilationW, etc need to be int_64?

Reply:
Because the call sites pass int_64 to them